{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Low"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright Â© 2023 Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An insecure use of a temporary file has been found in Python.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "Python is an interpreted, interactive, object-oriented programming\nlanguage.\n\nZack Weinberg discovered that os._execvpe from os.py in Python 2.2.1 and\nearlier creates temporary files with predictable names. This could allow\nlocal users to execute arbitrary code via a symlink attack\n\nAll users should upgrade to these errata packages which include a patch to\npython 1.5.2 to correct this issue.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHSA-2003:048",
				"url": "https://access.redhat.com/errata/RHSA-2003:048"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/security/updates/classification/#low",
				"url": "https://access.redhat.com/security/updates/classification/#low"
			},
			{
				"category": "external",
				"summary": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=156556",
				"url": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=156556"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2003/rhsa-2003_048.json"
			}
		],
		"title": "Red Hat Security Advisory: python security update",
		"tracking": {
			"current_release_date": "2003-02-07T00:00:00Z",
			"generator": {
				"date": "2023-06-30T18:59:00Z",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "3.18.0"
				}
			},
			"id": "RHSA-2003:048",
			"initial_release_date": "2003-02-12T14:39:00Z",
			"revision_history": [
				{
					"date": "2003-02-07T00:00:00Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"category": "product_family",
						"name": "Red Hat Advanced Products",
						"product": {
							"name": "Red Hat Advanced Products",
							"product_id": "Red Hat Advanced Products"
						}
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2002-1119",
			"discovery_date": "",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla",
					"text": "https://bugzilla.redhat.com/show_bug.cgi?id=1616838"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "os._execvpe from os.py in Python 2.2.1 and earlier creates temporary files with predictable names, which could allow local users to execute arbitrary code via a symlink attack.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "security flaw",
					"title": "Vulnerability summary"
				}
			],
			"product_status": {
				"fixed": [
					"Red Hat Advanced Products"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2002-1119",
					"url": "https://www.cve.org/CVERecord?id=CVE-2002-1119"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2002-1119",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2002-1119"
				},
				{
					"category": "external",
					"summary": "CVE-2002-1119",
					"url": "https://access.redhat.com/security/cve/CVE-2002-1119"
				},
				{
					"category": "external",
					"summary": "bz#1616838: CVE-2002-1119 security flaw",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1616838"
				}
			],
			"release_date": "2002-08-01T00:00:00Z",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "Before applying this update, make sure all previously released errata\nrelevant to your system have been applied.\n\nPlease note that this update is available via Red Hat Network.  To use Red\nHat Network, launch the Red Hat Update Agent with the following command:\n\nup2date\n\nThis will start an interactive process that will result in the appropriate\nRPMs being upgraded on your system.",
					"product_ids": [
						"Red Hat Advanced Products"
					],
					"url": "https://access.redhat.com/errata/RHSA-2003:048"
				}
			],
			"title": "CVE-2002-1119 security flaw"
		}
	]
}
